third_party/move/move-compiler-v2/tests/checking/variants/variants_parse_err4.exp

```diff
@@ -7,4 +7,4 @@ error: unexpected token
    │               ^^^^
    │               │
    │               Unexpected 'self'
-   │               Expected '('
+   │               Expected ';'
```

Comment:
Perhaps, but I do not see a feasible way to avoid this. Only `match (` is now recognized as a prefix for a potential match statement. Its actually logical because `match; self` can be valid code.